playbooks/gpu-clocks.yml

```diff
@@ -0,0 +1,15 @@
+---
+- hosts: slurm-node
+  become: true
+  tasks:
+   # - name: install custom facts
+   #   include_role:
+   #     name: facts
```

Comment:
I think you want to leave this un-commented out so that you can run it standalone, the facts role is what finds the GPUs on the system.

Reply:
Ah ok i didn't know that's what the facts role did:)